ci/jenkinsfiles/common-lib.groovy

```diff
@@ -97,10 +241,22 @@ String mavenArgs() {
   return args
 }
 
+boolean namespaceExists(String namespace) {
+  return sh(returnStatus: true, script: "kubectl get namespace ${namespace}") == 0
+}
+
+boolean needsPreviewCleanup() {
+  return this.isPullRequest() && !pullRequest.labels.contains('preview')
+}
+
 boolean needsSaucelabs() {
   return this.isPullRequest() && pullRequest.labels.contains('saucelabs')
 }
 
+void rolloutStatus(String kind, String name, String timeout, String namespace) {
+  sh "kubectl rollout status ${kind} ${name} --timeout=${timeout} --namespace=${namespace}"
+}
+
 void runBackEndUnitTests() {
   return {
```

Comment:
The return type is declared to be `void` but you return an object, perhaps you want to use `def` keyword